lib/actions/push.ts

```diff
@@ -98,30 +113,53 @@ async function getAppOwner(sdk: BalenaSDK, appName: string) {
 	return selected.extra;
 }
 
-export const push: CommandDefinition<
-	{
-		applicationOrDevice: string;
-	},
-	{
-		source: string;
-		emulated: boolean;
-		nocache: boolean;
+function parseRegistrySecrets(secretsFilename: string): RegistrySecrets {
+	let registrySecrets: RegistrySecrets = {};
+	if (secretsFilename) {
```

Comment:
I'd prefer this typed correctly above, e.g.
```
function parseRegistrySecrets(secretsFilename?: string): RegistrySecrets
```
and then a test like so below:
```
if (secretsFilename == null)
```

Note the `==`, rather than `===` which will test for both `undefined` and `null`

Reply:
Agreed. I've ended up moving the test to the caller, so the function must be called with a truthy filename.